tests/serpapi_test.ts

```diff
@@ -40,565 +40,637 @@ import {
 loadSync({ export: true });
 const SERPAPI_TEST_KEY = Deno.env.get("SERPAPI_TEST_KEY") ?? "";
 const HAS_API_KEY = SERPAPI_TEST_KEY.length > 0;
-const BASE_URL = Deno.env.get("ENV_TYPE") === "local"
-  ? "http://localhost:3000"
-  : "https://serpapi.com";
-
-describe("getAccount", {
-  sanitizeOps: false, // TODO(seb): look into how we can avoid setting these to false
-  sanitizeResources: false,
-}, () => {
-  let urlStub: Stub;
-
-  beforeAll(() => {
-    urlStub = stub(_internals, "getBaseUrl", () => BASE_URL);
-  });
-
-  afterEach(() => {
-    config.api_key = null;
-  });
-
-  afterAll(() => {
-    urlStub.restore();
-  });
-
-  it("with no api_key", () => {
-    assertRejects(
-      async () => await getAccount({ api_key: "" }),
-      MissingApiKeyError,
-    );
-    assertRejects(
-      async () => await getAccount({}),
-      MissingApiKeyError,
-    );
-    assertRejects(
-      async () => await getAccount(),
-      MissingApiKeyError,
-    );
-  });
+const BASE_OPTIONS = Deno.env.get("ENV_TYPE") === "local"
+  ? {
+    hostname: "localhost",
+    port: 3000,
+  }
+  : {
+    hostname: "serpapi.com",
+    port: 443,
+  };
```

Comment:
Let's include `protocol` here

```suggestion
  ? {
    hostname: "localhost",
    port: 3000,
    protocol: "http",
  }
  : {
    hostname: "serpapi.com",
    port: 443,
    protocol: "https",
  };
```